src/components/consul/selectors/ImageSelector.js

```diff
@@ -92,15 +100,58 @@ export const ImageSelector = ({ control, errorType, field, imageId, isMultiImage
     /* used to specify the mimeType when uploading to the server */
     const imageType = IMAGE_TYPE_REGEX.exec(uri)[1];
     const mimeType = `${type}/${imageType}`;
+    const uriSplitForImageName = uri.split('/');
+    const imageName = uriSplitForImageName[uriSplitForImageName.length - 1];
 
     errorTextGenerator({ errorType, infoAndErrorText, mimeType, setInfoAndErrorText, uri });
 
-    setImagesAttributes([...imagesAttributes, { uri, mimeType }]);
+    setImagesAttributes([...imagesAttributes, { uri, mimeType, imageName }]);
   };
 
   const values = jsonParser(value);
 
   if (isMultiImages) {
+    if (selectorType === IMAGE_SELECTOR_TYPES.SUE) {
```

Comment:
sue stuff inside of consul :o? could this be placed more clearly in different files or getting this out of consul?

Reply:
Yes. This is the component we use for consul and volunteer. But it's not a problem for both of these features. Here an extra control has been added so that sue has a different look. I think a component like `SueImageComponent` could be prepared to make this more descriptive.